tests/inventory.sample-crc-vagrant.yaml

```diff
@@ -0,0 +1,33 @@
+all:
+  vars:
+      install_yamls_path: /CUSTOMIZE_THIS
+      reset_crc_storage: true
+
+      oc_header: |
+        source CUSTOMIZE_THIS.sh
+      oc_login_command: |
+        oc login -u kubeadmin -p crcadmin
+
+      cinder_password: CUSTOMIZE_THIS
+      glance_password: CUSTOMIZE_THIS
+      ironic_password: CUSTOMIZE_THIS
+      neutron_password: CUSTOMIZE_THIS
+      nova_password: CUSTOMIZE_THIS
+      octavia_password: CUSTOMIZE_THIS
+      placement_password: CUSTOMIZE_THIS
```

Comment:
This bit could be improved upon, to make the tests themselves fetch the passwords from a Heat environment file for example. The procedure for that could be slightly different per environment though, e.g. it would differ slightly in Standalone envs vs. full Director envs, so we'd have to make the fetching mechanism either generic enough, or having a "switch" logic for env types.

So it depends where do we want the "password fetching" bit going forward, whether as part of the test suite, or test infra setup.

Reply:
So far i've only run against CRC+Vagrant env, once we see e.g. how would these passwords be extracted in CI context, it may become better apparent what could be the right way to abstract this in the test suite (assuming we'd want the extraction logic as part of the test suite).

Reply:
Based on the call discussion i'm moving the secrets into their own file, and i'll do the same for variables. This loses the advantage of having a single file with all environment info, but may be more suitable for CI, e.g. writing out the vars/secrets files via automation without having to watch out for YAML indentation in the inventory.